src/atoms/View/__tests__/__snapshots__/View.web.test.js.snap

```diff
@@ -3,10 +3,10 @@
 exports[`<View /> as renders View as div 1`] = `
 <div>
   <div
-    class="Viewweb__View-sc-1cphyjf-0 lhSUkp"
+    class="Viewweb__View-sc-1cphyjf-0 ljAogI"
```

Comment:
I think builds are failing because of this. Mismatch in the classname here.

Reply:
@krishnaacc123 You can update the test cases by adding `-u` after the script.` yarn test:web -u`